pczt/src/orchard.rs

```diff
@@ -140,6 +140,7 @@ pub struct Spend {
     ///
     /// This exposes the asset value to all participants. For Signers who don't need this
     /// information, or after signatures have been applied, this can be redacted.
+    #[getset(get = "pub")]
```

Comment:
why this is needed?

Reply:
To auto-generate public getter, it is used a lot in pczt, pretty convenient